design-patterns/gc-intro/samples/gc-intro.html

```diff
@@ -0,0 +1,3 @@
+<h1 property="name" id="wb-cont" dir="ltr">{{ page.title }}</h1>
```

Comment:
```suggestion
<h1 property="name" id="wb-cont">{{ page.title }}</h1>
```

It is already defined on the html element and that is an attribute that all sub element inherit